samples/nrf9160/modem_shell/src/location/location_shell.c

```diff
@@ -130,6 +132,7 @@ static struct option long_options[] = {
 	{ "gnss_cloud_pvt", no_argument, 0, LOCATION_SHELL_OPT_GNSS_LOC_CLOUD_PVT },
 	{ "cellular_timeout", required_argument, 0, LOCATION_SHELL_OPT_CELLULAR_TIMEOUT },
 	{ "cellular_service", required_argument, 0, LOCATION_SHELL_OPT_CELLULAR_SERVICE },
+	{ "cellular_count", required_argument, 0, LOCATION_SHELL_OPT_CELLULAR_COUNT },
```

Comment:
Correct naming would be `cellular_cell_count` but sounds a bit weird and long. I'm ready to change if consistency is preferred.

Reply:
I was going to propose `cellular_cell_count` before I read your comment. It is longer, but it is also more correct. To me `cellular_count` sounds a bit weird. 😃 But somebody else can also comment...

Reply:
If there is someone who thinks cellular_cell_count is fine, I'll change it so will do it :)